src/garage/experiment/experiment.py

```diff
@@ -240,11 +239,8 @@ def run_experiment(method_call=None,
             commands without executing them.
         env (dict): Extra environment variables.
         variant (dict): If provided, should be a dictionary of parameters.
-        use_tf (bool): Used along with the Theano and GPU configuration
-            when using TensorFlow
         use_gpu (bool): Whether the launched task is running on GPU.
-            This triggers a few configuration changes including certain
-            environment flags.
+            It assumes proper configuration of CUDA to use GPU.
```

Comment:
it seems to me that most people with GPUs will want this to default to True, whereas people with only CPUs needs this to default to False.

AFAICT is harmless for it to default to `True` for CPU-based machines, so perhaps the default should be `True`?

Also, what if this accepts either a `bool` or a `tuple`/`list`? If it is a `True`/`False` we use all or not CPUs available. If it is a collection, we interpret that as whitelist of GPUs we are allowed to use. @krzentner I'd like to hear your thoughts.